demonstrations/tutorial_multiclass_classification.py

```diff
@@ -90,14 +91,21 @@ def layer(W):
 # Data is embedded in each circuit using amplitude embedding:
 
 def circuit(weights, feat=None):
-    qml.templates.embeddings.AmplitudeEmbedding(feat, [0, 1], pad=0.0, normalize=True)
+    qml.templates.embeddings.AmplitudeEmbedding(feat, range(0,num_qubits), pad=(2**num_qubits-feature_size), normalize=True)
     for W in weights:
         layer(W)
-    return qml.expval(qml.PauliZ(0))
 
-qnode1 = qml.QNode(circuit, dev).to_torch()
-qnode2 = qml.QNode(circuit, dev).to_torch()
-qnode3 = qml.QNode(circuit, dev).to_torch()
+    even_wirelist = [i for i in range(num_qubits) if i%2 == 0]
+    PZ = qml.PauliZ(0)
+    for wi in even_wirelist[1:]:
+        PZ = PZ@qml.PauliZ(wi)
+    if(num_qubits%2!=0):
+        PZ = PZ@qml.PauliZ(num_qubits-1)
+    return qml.expval(PZ)
```

Comment:
Might be worth running `black -l 100 tutorial_multiclass_classification.py` on this file to pick up some formatting details here like adding whitespace (might change some further parts of the script, which is okay).

Reply:
I'm not sure how to do that, maybe because I'm new here!

Reply:
No worries there! `black` is a formatting tool for Python that reformats Python code such that the result conforms with PEP8 coding standards. We use that for developing PennyLane. In this case, it could help with adding some whitespace in the code.

When using `pip`, `black` can be installed by running `pip install black` from the command line. Once installed, the command `black` is available to be used. `black -l 100 tutorial_multiclass_classification.py` can then be run in the folder where the `tutorial_multiclass_classification.py` file is located.

You can find more details on submitting a Pull Request [here](https://pennylane.readthedocs.io/en/latest/development/guide/pullrequests.html).

Reply:
Thanks again for the explanation. I ran the tool and here is the result:
```suggestion
    even_wirelist = [i for i in range(num_qubits) if i % 2 == 0]
    PZ = qml.PauliZ(0)
    for wi in even_wirelist[1:]:
        PZ = PZ @ qml.PauliZ(wi)
    if num_qubits % 2 != 0:
        PZ = PZ @ qml.PauliZ(num_qubits - 1)
    return qml.expval(PZ)
```
!